src/vcConvert.cpp

```diff
@@ -421,7 +421,7 @@ void vcConvert_ShowUI(vcState *pProgramState)
         ImGui::Button(localizationBuffer, ImVec2(-1, 40));
       }
     }
-    else if (pSelectedJob->status == vcCQS_Completed)
+    else if (pSelectedJob->status == vcCQS_Completed || pSelectedJob->status == vcCQS_WriteFailed)
```

Comment:
This doesn't seem right...On a failure, it opens up the UI to "add to scene" and what not?

Should `vcCQS_WriteFailed` be with `vcCQS_Preparing` and `vcCQS_Cancelled` above?